setup.py

```diff
@@ -29,4 +37,11 @@
     url="https://github.com/fidelityInternational/djangocms-navigation",
     license="BSD",
     test_suite="tests.settings.run",
+    tests_require=TEST_REQUIREMENTS,
+    dependency_links=[
```

Comment:
Are the dependancy links for tests only or will they also affect normal install of the package by downloading the packages? So in theory you could have the packages in .egg and the site-packages directory?

Reply:
The dependency links works for both tests and `install_requires`. I've changed the install require to use djangocms 3.5, when you run the tests it will install djangocms 4.0 instead. 

Does that answer your question?

Reply:
It does, as long as that still works for the tests I would prefer to leave the install requirements as they were, if we changed that we would need to change the other addons too as they a have that requirement.